lab/requirements.md

```diff
@@ -0,0 +1,53 @@
+# OPI Lab Requirements
+
+## Access
+
+- Lab staff can access to all machines.
+- OPI members should access only to their own machines.
+- Remote access to a machine requires private VPN specific to the machine.
+- OPI members cannot access to a machine from another one.
+- SSH connection is available through private VPN.
+- Emergency console can be enabled for remote access in case of issue.
+
+## Backup
+
+- All servers must be backed up.
+- Restoration of servers must be done within a couple of hours.
```

Comment:
Again, we should not be treating these like pets, but like cattle.

Reply:
I agree that not all servers must be backed up.  I do say, though, that all critical servers must be backed up.  I would like to hear people's view on whether the set of critical servers is the empty set.

Reply:
All of our things should be ephemeral, and ideally we would store any long-lived data in something like S3.

Reply:
Ultimately, folks will need to store state.  If the lab uses S3 to store state, great!  That's the way it should be.  So long as S3 has an SLA that backs up that state and staffed with personnel to restore it within a reasonable time, I would say that would be a way to meet the "all critical servers must be backed up" requirement.